packages/core/test/lib/tracing/trace.test.ts

```diff
@@ -189,18 +169,57 @@ describe('startSpan', () => {
       expect(ref.spanRecorder.spans).toHaveLength(2);
       expect(ref.spanRecorder.spans[1].op).toEqual('db.query');
     });
+  });
 
-    it('forks the scope', () => {
-      const initialScope = getCurrentScope();
+  it('creates & finishes span', async () => {
```

Comment:
just moved these out of the `it.each()` as they aren't actually using any of it.